src/greynirseq/noising/ieg/errorrules/nouncase.py

```diff
@@ -0,0 +1,39 @@
+import random
+
+from ieg import b
+
+from .errors import ErrorRule
+
+
+class NounCaseErrorRule(ErrorRule):
```

Comment:
Er einhver ástæða fyrir að þessi villuregla er í sinni eigin skrá?

Reply:
The idea was to migrate the rules into separate files, but it's only been done for this one yet.